<?xml version="1.0"?>
<launch>
   <node name="rona_obstacle_detector_node" pkg="rona_obstacle" type="rona_obstacle_detector_node" output="screen">
      <param name="pub_obstacle_topic"      type="string" value="obstacle"       />
      <param name="sub_scan_topic"          type="string" value="scan"           />
      <param name="map_frame"               type="string" value="map"            />
      <param name="angle_min"               type="double" value="-1.3 "          />
      <param name="angle_max"               type="double" value="1.3  "          />
      <param name="y_min"                   type="double" value="-0.25"          />
      <param name="y_max"                   type="double" value="0.25 "          />
      <param name="save_dist"               type="double" value="0.4  "          />
      <param name="dist_offset"             type="double" value="-0.01"          />
      <param name="min_laser_points"        type="int"    value="15   "          />
      <param name="drop_laser_points_fac"   type="double" value="0.35 "          />
      <param name="min_obstacle_size"       type="double" value="0.3  "          />
   </node>
</launch>